lib/mvi/responses/id_parser.rb

```diff
@@ -12,28 +12,35 @@ class IdParser
       # {source id}^{id type}^{assigning facility}^{assigning authority}^{id status}
       # id type - NI = national identifier
       #           PI = patient identifier
-      # id status -  A = active
-      #              P = permanent
-      #              H = deprecated(do not use)
-      #              L = pending local merge (do not use)
-      #              PCE = ??
+
+      # id status -  A = Active
+      #              P = Permanent
+
+      # Should definitely NOT be using id_status
+      #              H = Deprecated due to local merge
+      #              D = Deprecated from a Duplicate
+      #              M = Deprecated from a Mismatch
+      #              U - Deprecated from an Unlink
+      #              L = pending local merge - This is a value identified for correlations including EDIPI when the
+      #                   identifier has been deprecated/associated to another active ID.
+      #              PCE = Pending Cat Edit correlations (unsure if this should be used, likely not)
 
       def parse(ids)
         ids = ids.map(&:attributes)
-
```

Comment:
probably want to [ignore whitespace](https://github.com/department-of-veterans-affairs/vets-api/pull/3259/files?w=1) when viewing the diff